#!/usr/bin/env python

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("waiting_time", double_t, 0, "time in miliseconds between two tones", 100., 0., 2000.)

exit(gen.generate("sound_publisher", "config_sound_publisher", "sound_publisher"))
